content/user-research/identifying-users-needs/non-digital.md

```diff
@@ -4,17 +4,17 @@ layout: text/textblock
 ### [2]Who to include in your research
 When you’re researching with users who don’t or can’t use digital services you should focus on the people who are current users of your service or who are likely to use it in future.
 
-Research with people who have the lowest levels of digital skill, confidence and internet access. This will help you understand people whose support needs are most difficult to meet (for example, they may not be able to leave their home or have no computer).
+Include people in your research who have low levels of digital skill and internet access. They will help you understand support needs that are the most difficult to meet. For example, some users may not be able to leave their homes or have no computer.
```

Comment:
What about '... or may not have a computer'? (I read 'or have no computer' as something they may not have -- as if it were a double negative). Might just be me though! :)